### PR TITLE
Meshing tutorial update

### PR DESCRIPTION
**Pull request by IvarStefansson**
Updates to the tutorial meshing_of_fractures. Primary motivation is accessibility of information, particularly w.r.t. how fractures are defined.

PR also contains a few minor changes elsewhere in the code, notably renaming of the method `_stress_tensor` to `_stiffness_tensor` in the models.

### CODE REVIEW

**Comment by IvarStefansson** [comment]
Please read the update to the ad tutorial carefully to make sure I haven't misunderstood something, @keileg. Also, please feel free to improve/elaborate/clarify this or related topics.

File: tutorials/meshing_of_fractures.ipynb [hunk 1]
```diff
@@ -205,12 +299,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We have not yet set a boundary for the `FractureNetwork`, and effectively for the domain. The boundary is defined as a box, and is imposed in the following way"
+    "We have not yet set a boundary for the `FractureNetwork`, and effectively for the domain. The boundary is defined as a box, and is imposed in the following way:"
```
Comment (keileg) [inline tutorials/meshing_of_fractures.ipynb:302]:
```suggestion
    "We have not yet set a boundary for the `FractureNetwork3d`, and effectively for the domain. The boundary is defined as a box, and is imposed in the following way:"
```

File: tutorials/meshing_of_fractures.ipynb [hunk 2]
```diff
@@ -267,20 +361,20 @@
...
     "\n",
-    "In PorePy, these issues are attempted resolved as follows: The `FractureNetwork` has an attribute `tol` that represent the geometric tolerance used in the calculation of intersections and subsequent splitting of the fractures. If meshing is done with gmsh, the tolerances used in PorePy and gmsh are related. The approach works reasonably well, but for complex configurations of fracture intersections, stability issues can arise. We are working to iprove these matters."
+    "In PorePy, these issues are attempted resolved as follows: The `FractureNetwork` has an attribute `tol` that represents the geometric tolerance used in the calculation of intersections and subsequent splitting of the fractures. If meshing is done with gmsh, the tolerances used in PorePy and gmsh are related. The approach works reasonably well, but for complex configurations of fracture intersections, stability issues can arise. "
```
Comment (keileg) [inline tutorials/meshing_of_fractures.ipynb:370]:
```suggestion
    "In PorePy, these issues are attempted resolved as follows: The `FractureNetwork3d` (and the corresponding `FractureNetwork2d`) has an attribute `tol` that represents the geometric tolerance used in the calculation of intersections and subsequent splitting of the fractures. If meshing is done with gmsh, the tolerances used in PorePy and gmsh are related. The approach works reasonably well, but for complex configurations of fracture intersections, stability issues can arise. "
```

**Review by keileg — APPROVED** [review]
A few minor suggestions. Thanks for the effort!

**Pull request merged**